ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -237,7 +241,7 @@ def dist_to_tl(self):
     def decelerate(self, waypoints):
         dist = self.dist_to_tl()
         if self.tl_idx > - 1 and dist < LOOKAHEAD_WPS:
-            prev_idx = max(dist-2, 0)
+            prev_idx = max(dist-4, 0)
```

Comment:
I think we should define this variable somewhere. It really has an important meaning: how many waypoints before the stop light we stop the car.